source/globals.d.ts

```diff
@@ -12,3 +26,11 @@ declare module 'select-dom' {
 	}
 	export = select;
 }
+
+declare module "webext-options-sync" {
```

Comment:
No types available for `webext-options-sync` package, so I added what was needed so far. I'll add more of the API in here if I see other methods are used in the codebase.